README.md

```diff
@@ -58,6 +58,30 @@ git clone --bare https://github.com/standardebooks/david-lindsay_a-voyage-to-arc
 
 If everything went well, `https://standardebooks.test/ebooks/` will show the one ebook you deployed.
 
+## Installation using Docker
+
+We provide a Dockerfile for testing code changes. You can build an image with:
+
+```shell
+docker build . -t standardebooks
+```
+
+Then run the built image with:
+
+```shell
+docker run -dp 443:443 -v "$(pwd):/standardebooks.org/web" standardebooks:latest
+```
+
+The site will now be available at `https://localhost/`, although as it’s a self-signed certificate you’ll need to accept whatever browser security warnings come up.
+
+If you make a change to the PHP and want to see that reflected in the site, then you’ll need to restart PHP. Connect to your container and run:
```

Comment:
I don't think this is correct, generally FPM doesn't need to be restarted for PHP changes to take effect. It *may* need to be restarted if data cached in APCu is changed and the c cache has to be flushed, but that's a different case than just "any PHP code changes."

Reply:
Actually, I think it depends on whether you're installing Ubuntu Desktop or Ubuntu Server. Desktop is configured with the opcache disabled and server is configured with it enabled. So if this specifies the server image then this would be correct after all.

Reply:
Aha, yep, that fixed it. https://github.com/standardebooks/web/pull/108/commits/a66a8c5e853705843ae67725593076a8a172e7dd